### PR TITLE
NotificationServer: Add a configurable notification timeout

### PR DESCRIPTION
**Pull request by metalvoidzz**

### CODE REVIEW

File: Services/NotificationServer/NotificationWindow.cpp [hunk 1]
```diff
@@ -90,7 +91,8 @@ NotificationWindow::NotificationWindow(const String& text, const String& title,
     widget.layout()->set_margins({ 8, 8, 8, 8 });
     widget.layout()->set_spacing(6);
 
-    if (icon.is_valid()) {
+    auto config = Core::ConfigFile::get_for_system("NotificationServer");
+    if (icon.is_valid() && config->read_num_entry("Window", "ShowIcons", true)) {
```
Comment (linusg) [inline Services/NotificationServer/NotificationWindow.cpp:95]:
```suggestion
    if (icon.is_valid() && config->read_bool_entry("Window", "ShowIcons", true)) {
```

**Review by linusg — CHANGES_REQUESTED** [review]

File: Services/NotificationServer/NotificationWindow.cpp (hunk 1 above)
Comment (metalvoidzz) [reply Services/NotificationServer/NotificationWindow.cpp:95]: Thanks, overlooked that one

**Comment by awesomekling** [comment]
This doesn't seem like a good default behavior to me.
If I went away from my computer for more than seconds, I still want to see my notifications when I return.

**Comment by metalvoidzz** [comment]
Right, maybe we should have some kind of notification history applet, similar to clipboard history. It could change icons to tell you about unread notifications.

**Comment by bcoles** [comment]
> 
> 
> Right, maybe we should have some kind of notification history applet, similar to clipboard history. It could change icons to tell you about unread notifications.

I like the idea of some sort of notification log. I also like the idea of having notifications disappear automatically.

Perhaps a notification applet icon could be displayed, with different icons depending on whether there are "new" notifications.

**Comment by linusg** [comment]
@metalvoidzz do you still intend to make any changes here? Also, the PR has some merge conflicts that need to be resolved.

**Pull request closed without merging**